utils/db.ts

```diff
@@ -128,7 +120,9 @@ export async function deleteItem(item: Item) {
 }
 
 export async function getItem(id: string) {
-  return await getValue<Item>(["items", id]);
+  const itemsByIdKey = ["items", id];
+  const res = await kv.get<Item>(itemsByIdKey);
+  return res.value;
```

Comment:
```suggestion
  const res = await kv.get<Item>(["items", id]);
  return res.value;
```
We can just directly use the key in the argument here.